src/test/kotlin/com/bardsoftware/papeeria/backend/cosmas/CosmasInMemoryServiceTest.kt

```diff
@@ -142,4 +142,72 @@ class CosmasInMemoryServiceTest {
                 .build()
         this.service.createVersion(newVersionRequest, createVersionRecorder)
     }
+
+    private fun addPatchToService(text: String, user: String = "No name", fileId: String = "0",
+                                            time: Long = 0) {

```

Comment:
Are you using proportional font by any chance? That could explain such strange indents. If you do, please change it to monospaced.